### PR TITLE
aio exc app webmodule

### PR DESCRIPTION
**Pull request by purplecabbage**
## Description
this restructures some of our existing code that we have been including in all app templates
front-end:
We were including a utils.js file which exported `actionWebInvoke`
This can be accomplished now by changing
`import actionWebInvoke from '../utils'`
to
`import { actionWebInvoke } from '@adobe/aio-exc-app/web'`

Similarly, in the action code we generate, we can remove utils from the template, and after adding a dependency, we can change the generated action code:
`const { errorResponse, getBearerToken, stringParameters, checkMissingRequestInputs } = require('../utils')`
becomes
`const { errorResponse, getBearerToken, stringParameters } = require('@adobe/aio-exc-app/action')`

## Related Issue

https://github.com/adobe/generator-aio-app/issues/130
ACNA-1041

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: packages/aio-exc-app/action/index.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+   *
+   * @param {object} params action input parameters.
+   *
+   * @returns {string|undefined} the token string or undefined if not set in request headers.
+   *
+   */
+  function getBearerToken (params) {
+    if (params.__ow_headers &&
+        params.__ow_headers.authorization &&
+        params.__ow_headers.authorization.startsWith('Bearer ')) {
```
Comment (shazron) [inline packages/aio-exc-app/action/index.js:107]: we can use optional chaining now :)

**Review by moritzraho — APPROVED** [review]

File: packages/aio-exc-app/action/index.js (hunk 1 above)
Comment (purplecabbage) [reply packages/aio-exc-app/action/index.js:107]: good point

**Comment by purplecabbage** [comment]
this needs tests before moving forward with the rest

**Comment by purplecabbage** [comment]
wish there was a way to trigger that there are now workflows to run

**Pull request merged**